docs/config-file-resolutions-yml.md

```diff
@@ -0,0 +1,94 @@
+# The `resolutions.yml` file
+
+Resolutions allow *resolving* any errors or policy violations by marking them as acceptable and can be ignored.
+
+You can re-use [examples/resolutions.yml](examples/resolutions.yml) as the base file for your scans.
+
+### When to Use Resolutions
+Resolutions should be used when it impossible to solve an issue or is planned to be fixed at a later time.
+You can re-use [./examples/resolutions.yml](examples/resolutions.yml) as the base file for your resolutions.
+
+The sections below explains how to create resolutions in the `resolutions.yml` file
+which if passed as an argument to the `reporter` apply to each scan made. If a resolution is project-specific then add it in the [.ort.yml](config-file-ort-yml.md) file for the project.
+
+Resolutions are only taken into account the `reporter`, while the `analyzer` and `scan` results ignore them.
+
+## Resolution Basics
+
+A resolution is applied to specific errors or violations via the regular expression specified
+in the `message` of a resolution.
+
+To be able to show why a resolution is acceptable, each resolution must include an explanation. 
+The explanation consists of:
+
+* `reason` -- quick indicator why the issue was resolved. Must be selected from a predefined list of options. 
+* `comment` -- free text that provides an additional explanation and maybe a link for further information
+
+## Resolving Errors
+
+If the ORT results show errors, the best approach is usually to fix them and run the scan again. 
+However, sometimes it is not possible, for example if error occurs in the license scan
+of a third-party dependency which cannot be fixed or updated.
+
+In such situations, you can *resolve* the error in any future scan by adding a resolution
+to the `resolutions.yml` to mark it as acceptable.
+
+The code below shows the structure of an error resolution in the `resolutions.yml` file:
+
+```yaml
+errors:
+- message: "A regular expression matching the error message."
+  reason: "One of ErrorResolutionReason e.g. BUILD_TOOL_ISSUE,CANT_FIX_ISSUE,SCANNER_ISSUE."
+  comment: "A comment further explaining why the reason above is acceptable."
+```
+where the list of available options for `reason` is defined in
+[ErrorResolutionReason.kt](../model/src/main/kotlin/config/ErrorResolutionReason.kt)
+
+For example, to ignore an error related to a build tool problem, your `resolutions.yml` could include:
+
+```yaml
+errors:
+- message: "Does not have X.*"
+  reason: "BUILD_TOOL_ISSUE"
+  comment: "Error caused by a known issue for which fix is being implemented, see https://github.com/..."
+```
+
+## Resolving Policy Rule Violations
+
+Resolutions should not be not used to resolve license policy violations as they do not change generated open notices.
+To resolve a license policy violation either add a local license finding curation
```

Comment:
license finding curation --> license curation

Reply:
Done, change it to `license_findings` curation refering to name of variable in .ort.yml